openbb_terminal/stocks/behavioural_analysis/ba_controller.py

```diff
@@ -717,6 +717,15 @@ def call_sentiment(self, other_args: List[str]):
             default=6,
             help="number of days in the past to extract tweets.",
         )
+        parser.add_argument(
+            "-c",
+            "--compare",
+            action="store",
```

Comment:
For true/false flags you can use here "store_true" as action and remove both the type and the default

Reply:
That's good advice :)